packages/cloud_firestore/cloud_firestore/test_driver/cloud_firestore_e2e_test.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
```

Comment:
move this file to 
`cloud_firestore/cloud_firestore/exmaple/test_driver/cloud_firestore_e2e_test.dart`
to fix CI